benchmark/benchmark_serving.py

```diff
@@ -26,6 +25,31 @@ def get_launching_server_cmd(model_path, backend, server_config):
     return cmd
 
 
+def get_output_file(model_path, backend, server_config):
+    """Generate the benchmark output filename."""
+    model_name = server_config.get('model_name', None) or os.path.basename(model_path)
+
+    if backend not in ['turbomind', 'pytorch', 'sglang', 'vllm']:
+        raise ValueError(f'Unknown backend: {backend}')
+
+    if backend in ['sglang', 'vllm']:
+        return f'benchmark_{model_name}_{backend}.csv'
+
+    # For turbomind/pytorch backends
+    params = {
+        'bs': server_config['max_batch_size'],
+        'tp': server_config.get('tp', 1),
+        'cache': server_config.get('cache_max_entry_count', 0.8),
+        'mptk': server_config.get('max_prefill_token_num', ''),
+    }
+    params_str = '_'.join(f'{k}{v}' for k, v in params.items() if v != '')
```

Comment:
It should be a bit better if we make `params.items()` sorted. It will keep the filename consistent across different environments.